Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
 
 name = "conduit-static"
-version = "0.8.1"
+version = "0.9.0-alpha.0"
 authors = ["wycats@gmail.com",
            "Alex Crichton <alex@alexcrichton.com>"]
 description = "Middleware for serving static files for conduit"
 repository = "https://github.com/conduit-rust/conduit-static"
 license = "MIT"
 
 [dependencies]
-conduit = "0.8"
+conduit = "0.9.0-alpha.0"
 conduit-mime-types = "0.7"
 time = "0.1"
```

Comment:
`time` is now v0.2 but many crates still rely on v0.1 so it's fine, I think (in crates.io deps, `cookie` v0.13.0 only relies on v0.2).

Reply:
Good catch.  I plan to bump the `cookie` dependency as well.  I'll open an issue to track bumping `time` since it looks like it will require some code changes.